examples/django/sample_project/README.md

```diff
@@ -0,0 +1,83 @@
+# pygeoapi Django integration
+
+This directory contains a [sample Django project](https://djangoproject.com) demonstrating how to
+integrate pygeoapi into your Django application.
+
+Django is a Python web framework that encourages rapid development and clean, pragmatic design.
+
+In this document we create a sample Django project and use pygeoapi as a pluggable, embedded application.
+
+To create your Django application from scratch follow these steps: 
+
+```bash
+
+# create a project directory and create a fresh virtual environment
+python3 -m venv env
+cd env
+source bin/activate
+
+# install dependencies
+pip install Django pygeoapi
+
+# create a Django project
+django-admin startproject sampleproject
+cd sampleproject
+
+# set pygeoapi environment variables
+export PYGEOAPI_CONFIG=`pwd`/pygeoapi-config.yml
+export PYGEOAPI_OPENAPI=`pwd`/example-openapi.yml
+
+# Django: collect all static assets/files
+python3 manage.py collectstatic
+
+# generate OpenAPI document
+pygeoapi openapi generate $PYGEOAPI_CONFIG --output-file $PYGEOAPI_OPENAPI
+```
+
+Update `settings.py`:
+
+```python
+
+import os
+from pygeoapi.django_app import config
+
+INSTALLED_APPS = [
+    # other apps
+    ....
+    #pygeoapi app
+    'pygeoapi'
+]
+
+# Put following setting after STATIC_URL 
+STATIC_ROOT = os.path.join( BASE_DIR / 'assets')
+
+# Specific pygeoapi setting
+PYGEOAPI_CONFIG = config()
+   ...
+```
+
+Update `urls.py` to run pygeoapi at e.g. `pga` path
+
+```python
+
+from django.contrib import admin
+from django.urls import path, include 
+from pygeoapi.django_pygeoapi import urls 
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('pga/', include(urls)) # added here
```

Comment:
I would use `oapi` as replacement of `pga`, accordingly with the `running` documentation page

Reply:
done